projects/console-web-server/src/ws/common/ws-common.service.ts

```diff
@@ -209,4 +212,18 @@ export class WsCommonService {
 
     return { result: true, resultCode: 1000, message: 'success' };
   }
+
+  public async validateHostWithWebsocket(
+    organizationId: OrganizationId,
+    hostId: HostId,
+    deviceId: DeviceId,
+    incomingMessage: IncomingMessage,
+    type: HOST_ACTION_TYPE,
+  ): Promise<HostPayload | null> {
+    const authHeader = incomingMessage.headers.authorization;
```

Comment:
@yowpark  If the input type is a project, I think it would be good to throw an Internal Error